addons/hw_posbox_homepage/views/upgrade_page.html

```diff
@@ -30,27 +30,26 @@
                     $('.message-title').text('IoTBox perform a self flashing it take a lot of time (~30min).');
                     $('.message-status').text('Prepare space for IoTBox.');
                     try {
-                        await $.ajax({url: '/hw_proxy/perform_flashing_create_partition/'}).promise();
-                        $('.message-status').text('Prepare new boot partition.');
-                        await $.ajax({url: '/hw_proxy/perform_flashing_download_raspios/'}).promise();
-                        $('.message-status').text('Download file for new boot partition.');
-                        await $.ajax({url: '/hw_proxy/perform_flashing_copy_raspios/'}).promise();
-                        $('.message-status').text('Prepare to restart and installation of the new version of the IoT Box.');
+                        $('.message-status').text('Preparing the new boot partition.');
+                        await $.ajax({url: '/hw_proxy/perform_flashing/create_partition'}).promise();
+                        $('.message-status').text('Downloading files for the new boot partition.');
+                        await $.ajax({url: '/hw_proxy/perform_flashing/download_raspios'}).promise();
+                        $('.message-status').text('Preparing to restart and install the new version of the IoT Box.');
+                        await $.ajax({url: '/hw_proxy/perform_flashing/copy_raspios'}).promise();
```

Comment:
- Message were not showing in the right order (they were showing after the action they were describing was performed),
- tiny refactor of the calls to the controller: now one only route to perform all the actions.